src/utils/Crc.cxxtest

```diff
@@ -175,3 +175,25 @@ TEST(Crc8Test, Collision)
     LOG(INFO, "s1=%s s2=%s", string_to_hex(s1).c_str(),
         string_to_hex(s2).c_str());
 }
+
+TEST(Crc8Test, LogonExample)
```

Comment:
I'm a bit confused. These tests, as well as some above, don't have any "EXPECTs". What is the acceptance criteria? Doesn't crash?

Reply:
added comment:
```
// These examples contain specific messages from other unittests and print
// their CRC. This pattern can be used for developing other tests. There are no
// expectations in these tests.
```